05_book_titles/book.rb

```diff
@@ -1,3 +1,33 @@
+# ok so i need a list of 'small words' that won't be capitalized
+# i think this can just be local to the book class as it is
+
 class Book
```

Comment:
Let's add a `setter` and `getter`. Inside the `getter`, it will call the new method  `titleize`. And then inside the setter we can retrieve the title.